dclab/rtdc_dataset/feat_temp.py

```diff
@@ -58,10 +59,11 @@ def set_temporary_feature(rtdc_ds, feature, data):
     Parameters
     ----------
     rtdc_ds: dclab.RTDCBase
-        Dataset for which to set the feature. Note that temporary
-        features cannot be set for hierarchy children and that the
+        Dataset for which to set the feature. Note that the
         length of the feature `data` must match the number of events
-        in `rtdc_ds`.
+        in `rtdc_ds`. If the dataset is a hierarchy child, the data will also
+        be set in the parent dataset, but only for those events that are part
+        of the child. All non-child events become np.nan.
```

Comment:
Please replace

"All non-child events become np.nan."

with

"For all events in the parent dataset that are not part of the child dataset, the temporary feature is set to `np.nan`."

Reply:
Yes, that's clearer.